db/schema.rb

```diff
@@ -638,6 +638,7 @@
     t.boolean  "exporting_photos",                                 default: false
     t.string   "color_theme",                        limit: 255
     t.boolean  "post_default_public",                              default: false
+    t.boolean  "chat_auto_login",                                  default: false
```

Comment:
Hm I personally feel like the default should be `true`. That's the reason we have so few users on the chat at the moment, they don't get they have to log in. So let's log them in by default, they can switch that off if they don't want to.